src/ReactTags.js

```diff
@@ -274,7 +274,13 @@ class ReactTags extends Component {
     e.preventDefault();
 
     const clipboardData = e.clipboardData || window.clipboardData;
-    const pastedText = clipboardData.getData('text');
+    const clipboardText = clipboardData.getData('text');
+
+    // Checking if the pasted text is greater than maxLength props
```

Comment:
This comment can be removed as the `if` condition which was previously added has been removed.